docs/video/ffmpeg/upscaling.md

```diff
@@ -6,11 +6,11 @@ icon: fontawesome/solid/arrow-up-right-dots
 # Why bother upscaling for YouTube?
 
 
-Scaling your video to twice it's resolution (e.g `1080p` :material-arrow-right: `2160p` / 4K) tricks YouTube into giving your video more bitrate. This is only useful for YouTube, 
+Scaling your video to a larger resolution (e.g `1080p` :material-arrow-right: `3840x2160` / 4K) tricks YouTube into giving your video more bitrate.
```

Comment:
i mean stay consistent and replace 1080p by 1920x1080 as well

Reply:
true